ext/jsonnet/vm.c

```diff
@@ -255,6 +257,112 @@ vm_set_max_trace(VALUE self, VALUE val)
     return Qnil;
 }
 
+static VALUE
+vm_set_fmt_indent(VALUE self, VALUE val)
+{
+    struct jsonnet_vm_wrap *vm = rubyjsonnet_obj_to_vm(self);
+    jsonnet_fmt_indent(vm->vm, NUM2INT(val));
+    return val;
+}
+
+static VALUE
+vm_set_fmt_max_blank_lines(VALUE self, VALUE val)
+{
+    struct jsonnet_vm_wrap *vm = rubyjsonnet_obj_to_vm(self);
+    jsonnet_fmt_max_blank_lines(vm->vm, NUM2INT(val));
+    return val;
+}
+
+static VALUE
+vm_set_fmt_string(VALUE self, VALUE val)
+{
+    char *str;
+    struct jsonnet_vm_wrap *vm = rubyjsonnet_obj_to_vm(self);
+    str = StringValueCStr(val);
+    if (str[0] == '\0' || str[1] != '\0' || (str[0] != 'd' && str[0] != 's' && str[0] != 'l')) {
```

Comment:
Should define Ruby-level constants for these values

Reply:
Like this? https://github.com/yugui/ruby-jsonnet/pull/17/commits/e47a1a3c75ac8a625e671ffb2dcf43d2ed3717cb